tests/api.c

```diff
@@ -29200,8 +29200,20 @@ static int test_wolfSSL_GetLoggingCb (void)
             wolfSSL_SetLoggingCb(NULL);
         }
     }
-    printf(resultFmt, ret == 0 ? passed : failed);
 #endif
+    if (ret == 0) {
+        if (wolfSSL_GetLoggingCb() == NULL) {
+            ret = 0;
+        }
+        if (ret == 0) {
+            if (wolfSSL_GetLoggingCb() != NULL) {
```

Comment:
This second wolfSSL_GetLoggingCb() check looks redundant to the one above it.  Let's leave the first one and remove this second one, thanks.